lib/dpul_collections/indexing_pipeline/figgy/resource.ex

```diff
@@ -93,7 +107,7 @@ defmodule DpulCollections.IndexingPipeline.Figgy.Resource do
   defp to_map(
          resource = %__MODULE__{metadata: %{"state" => [state], "visibility" => [visibility]}}
        )
-       when state != "complete" or visibility != "open" do
+       when state not in ["complete", "all_in_production"] or visibility != "open" do
```

Comment:
OH, interesting. Presumably converting the box hit this because the box has state? I think we still want the box number even if it's not all in production or complete. I wonder if this guard should be more strict to just EphemeraFolders?

Reply:
Done